src/index.html

```diff
@@ -7,7 +7,283 @@
+          </ul>
+        </nav>
+        <a class="menu__basket">
+          <img src="./images/basket.png" alt="basket">
+        </a>
+      </div>
+      <div class="header__essential">
+        <div class="header__heading">
+          <h1 class="header__title">Brand of eco-cosmetics <br> and magic candles</h1>
+          <a class="header__sale" href="#shop">20% of sales at a wild bird shelter</a>
+          <a class="header__button button" href="#shop">Shop</a>
+        </div>
+        <img
+          class="header__img"
+          src="./images/header.png"
+          alt="products"
+        >
+      </div>
+    </header>
+    <main class="main">
+      <section class="section">
+        <div class="section__info info">
+          <p class="info__text">Biologists say that birds are indicators of the environmental situation, and over the past half century, the population of seabirds has declined by 70%. Birds die due to environmental problems and human intervention in the natural environment.</p>
+          <h3 class="info__title">if you choose the gifts of nature and care about it’s - scroll down</h3>
+        </div>
+      </section>
+
+      <section class="section">
+        <p class="section__lateral lateral">witch of mountain winds</p>
+        <div class="section__about-brand about-brand">
+          <div class="about-brand__text">
+            <h2 class="about-brand__heading heading" id="about-brand">about our brand</h2>
+            <p>Our brand cosmetics are made from natural ingredients and we are promoting them to the masses. Shampoo, hair conditioner, soap - in recycled paper. Creams, oils, washing products in glass containers with recycled paper labels. No synthetics and microplastics in cosmetics. And the packaging from it can be returned for recycling or reused.</p>
+            <p class="about-brand__paragraph">We are all neighbors of one big House. And we must protect what we have. Destroying everything around for the sake of consumerism and imaginary convenience, we destroy what does not belong to us. Be aware of your choice!</p>
+            <a class="about-brand__link link" href="#creators">Learn More</a>
+          </div>
+          <div class="about-brand__photo">
+            <img
+              class="about-brand__img-big"
+              src="./images/about-big.jpg"
+              alt="candle"
+            >
+            <img
+              class="about-brand__img-small"
+              src="./images/about-small.jpg"
+              alt="small candle"
+            >
+          </div>
+        </div>
+      </section>
+
+      <section class="section">
+        <p class="section__lateral lateral">shop</p>
+        <div class="shop">
+          <h2 class="shop__heading heading" id="shop">shop</h2>
+          <nav class="shop__nav nav-shop">
+            <ul class="nav-shop__list">
+              <li class="nav-shop__item nav-shop__item-active"><a class="nav-shop__link nav-shop__link-active" href="#face">Face</a></li>
```

Comment:
```suggestion
              <li class="nav-shop__item nav-shop__item--active"><a class="nav-shop__link nav-shop__link-active" href="#face">Face</a></li>
```
если это модификатор то `--` или `_`